nekoyume/Assets/_Scripts/Game/Entrance/RoomEntering.cs

```diff
@@ -81,13 +83,32 @@ private IEnumerator Act()
             yield return new WaitForSeconds(1.0f);
             Widget.Find<Status>().Show();
             Widget.Find<EventBanner>().Show();
-            Widget.Find<EventReleaseNotePopup>().Show();
             var headerMenu = Widget.Find<HeaderMenuStatic>();
             if (!headerMenu.isActiveAndEnabled)
             {
                 headerMenu.Show();
             }
 
+            var worldInfo = States.Instance.CurrentAvatarState?.worldInformation;
+            if (worldInfo is not null)
+            {
+                var clearedStageId = worldInfo.TryGetLastClearedStageId(out var id) ? id : 1;
+
+                var patrolRewardPopup = Widget.Find<PatrolRewardPopup>();
+                if (patrolRewardPopup.CanClaim &&
+                    clearedStageId > PatrolRewardPopupLimitClearedStageId)
```

Comment:
```suggestion
                    clearedStageId >= PatrolRewardPopupLimitClearedStageId)
```
I think you intended this way. Is it correct?

Reply:
https://github.com/planetarium/NineChronicles/pull/3211/files#diff-4583ce3c7dbe246d3abd6d63da547b0e0da458f9d56729ec6592373fa4fba3f4R106
same vibe...